include/fmt/chrono.h

```diff
@@ -362,8 +362,10 @@ template <typename Rep, typename Period, typename Char>
 struct formatter<std::chrono::duration<Rep, Period>, Char> {
  private:
   align_spec spec;
+  core_format_specs pspec;
```

Comment:
We only need `int precision` instead of the whole `core_format_specs` here.

Reply:
That was my original implementation. I switched over to `core_format_specs` because of the `has-precision` predicate defined there.